include/itkTBBImageToImageFilter.hxx

```diff
@@ -304,8 +304,18 @@ template< typename TInputImage, typename TOutputImage >
 template<typename TInputImage, typename TOutputImage>

```

Comment:
The definition of the function shouldn't be between`#ifdef ITK_USE_TBB` `#endif` line 303 (sorry i probably did it first but it was a mistake)